File: abc138/abc138_b.cs
// https://atcoder.jp/contests/abc138/submissions/7037740

using System;
using System.Linq;

class Program
{
    static void Main()
    {
        var N = int.Parse(Console.ReadLine());
        var A = Console.ReadLine().Split(' ').Select(x => int.Parse(x)).ToArray();
        var result = A.Select(x => 1.0 / x).Sum();
        Console.WriteLine($"{1 / result}");
    }
}

File: abc/abc138/abc138_b.fs
// https://atcoder.jp/contests/abc138/submissions/11992427

let N = stdin.ReadLine() |> int

stdin.ReadLine().Split()
|> Array.map float
|> Array.sumBy (fun x -> 1.0 / x)
|> fun x -> 1.0 / x
|> stdout.WriteLine
